packages/plans-grid-next/static/mockServiceWorker.js

```diff
@@ -0,0 +1,281 @@
+/* eslint-disable */
```

Comment:
I'm seeing this warning in the console:
<img width="579" alt="CleanShot 2024-05-03 at 17 11 15@2x" src="https://github.com/Automattic/wp-calypso/assets/1842898/0a78b1a1-0847-4026-98d1-457d913b5f54">

Given the instruction and [the latest 2.0 migration note](https://mswjs.io/docs/migrations/1.x-to-2.x), I'm wondering if that means there is a way to configure it as a dependent package rather than a static asset here. It'd be wonderful if the former is possible, but definitely no need to change it now :)

Reply:
Interesting! I wonder why I'm not seeing that message. Regardless, generating `mockServiceWorker.js` dynamically sounds like a good idea. I've pushed an update to implement this: https://github.com/Automattic/wp-calypso/pull/89837/commits/c758283a418576140be37495c96ed482b3b1032b

Reply:
Would you mind testing this update @southp? If you remove `./static/mockServiceWorker.js` and then run `yarn workspace @automattic/plans-grid-next storybook`, it should be regenerated.